pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -93,12 +93,8 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 		},
 	}
 	flags := cmd.Flags()
-
-	releaseVer, err := version.ParseGitVersion(version.Get().GitVersion)
-	if err != nil {
-		klog.Infof("No default release version found. build version: %s", version.Get().String())
-		releaseVer = &version.ReleaseVersion{} // initialize to avoid panic
-	}
+	// image registry
+	flags.StringVarP(&opts.ImageRegistry, "image-registry", "", "", "Image registry. Choose a container registry to pull control plane images from")
```

Comment:
From the user's point of view, I don't know if this flag means `init` will download all images(for both `karmada's and Kubernetes's`). 
Can you give more detail info for this flag?

Reply:
Add `Tips: Reference kubeadm, including all images required to karmada init` ？

Reply:
I tend to one flag for all images, including both kube's and karmada's.
like `--private-image-registry` or `local-image-registry`.

Reply:
@lonelyCZ @carlory Please take a look

Reply:
`--private-image-registry` is better than local-image-registry.

Reply:
@lonelyCZ How do you say, can we add the `--private-image-registry` flag to specify a private image registry and once it is set, all images including karmada's and kube's will be pulled from that.

Reply:
> How do you say, can we add the --private-image-registry flag to specify a private image registry and once it is set, all images including karmada's and kube's will be pulled from that.

I think it is ok. To be precise, I just tested it that can set kube images by `--kube-image-registry` or ` --kube-image-mirror-country=cn` to override `--private-image-registry`. 

Like
```
./karmadactl init --image-registry=daocloud.io/atsctoo --kube-image-mirror-country=cn
```
It is great that `--kube-image-registry` and ` --kube-image-mirror-country=cn` take precedence over `--private-image-registry`

Reply:
ok, it seems that `-private-image-registry` is liked by more people. Does this parameter need more description? @RainbowMango

Reply:
Yes, we should clearly describe it at usage and the relationship between `--kube-image-registry` and `--kube-image-mirror-country=cn`.